schema/CMakeLists.txt

```diff
@@ -13,7 +13,9 @@ target_link_libraries(schema
     idl
     Seastar::seastar
     xxHash::xxhash
-    absl::headers)
+    absl::headers
+  PRIVATE
+    alternator)
```

Comment:
This should be merged since they're the same thing. But I think it's heading in the wrong direction and we should keep the boundary between core and alternator.